cmd/pkger/cmds/pack.go

```diff
@@ -123,7 +123,7 @@ func (e *packCmd) Flags() *flag.FlagSet {
 	return e.FlagSet
 }
 
-func Package(out string, decls parser.Decls) error {
+func Package(info here.Info, out string, decls parser.Decls) error {
```

Comment:
The bug is on line 122 where it's incorrectly doing `filepath.Dir` and jumping up one level. I, however, prefer your solution. 😄 

Can you delete the line 122 and and change `c.Name` to `info.Name` on line 126?

Thanks!

Reply:
I tried what you suggested but the generated `pkged.go` has a wrong package directive `package main` instead of `package pkged`

Reply:
I may have misunderstood what you asked, feel free to push some changes on the branch.

Reply:
That was my mistake, you're 100% correct. :) I mis-understood what was happening there.